docs/build-repro.md

```diff
@@ -23,6 +23,15 @@ The structure is base64 encoded and has the following format when decoded:
 
 ```json
 {
+  "frontend": "dockerfile.v0",
+  "attrs": {
+    "build-arg:foo": "bar",
+    "cmdline": "crazymax/dockerfile:master",
```

Comment:
have you tried rebuilding these options, eg. in buildctl. I think it shouldn't work atm as for example `cmdline` should disable the `#syntax` processing. https://github.com/moby/buildkit/blob/master/frontend/dockerfile/builder/build.go#L374-L389 . A possibility would be to filter these out, or convert to a gateway frontend.

Reply:
not tried with `buildctl`. Will take a look.